Framework/Core/src/CommonDataProcessors.cxx

```diff
@@ -327,10 +351,11 @@ DataProcessorSpec
     OutputInputs,
     Outputs{},
     AlgorithmSpec(writerFunction),
-    {{"res-file", VariantType::String, "AnalysisResults", {"Name of the output file"}},
-     {"res-mode", VariantType::String, "RECREATE", {"Creation mode of the result file: NEW, CREATE, RECREATE, UPDATE"}},
-     {"ntfmerge", VariantType::Int, 10, {"number of time frames to merge into one file"}},
-     {"keep", VariantType::String, "", {"Comma separated list of ORIGIN/DESCRIPTION/SUBSPECIFICATION to save in outfile"}}}};
+    {{"json-file", VariantType::String, "", {"Name of the json configuration file"}},
+     {"res-file", VariantType::String, "", {"Name of the output file"}},
+     {"res-mode", VariantType::String, "", {"Creation mode of the result files: NEW, CREATE, RECREATE, UPDATE"}},
+     {"ntfmerge", VariantType::Int, -1, {"Number of time frames to merge into one file"}},
+     {"keep", VariantType::String, "", {"Comma separated list of ORIGIN/DESCRIPTION/SUBSPECIFICATION:treename:col1/col2/..:filename"}}}};
```

Comment:
I would still provide sensible default values for all.

Reply:
I removed the default values to be able to check for whether an option has been set or not. Is there a better way to do that?

Reply:
Mmm... good point. I need to check I think there is but I am not sure.